File: src/UVa/11727/11727.cpp

/*
*****************************************************************
*                           PROBLEMA                            *
*****************************************************************
*                    UVa 11727 - Cost Cutting                   *
*****************************************************************
*/

#include <cstdio>

int main(){

	int n, a, b, c;
	scanf("%d",&n);

	for(int i=0; i<n; i++){

		scanf("%d %d %d",&a,&b,&c);
		if((a>b && a<c) || (a>c && a<b)) printf("Case %d: %d\n",i+1,a);
		if((b>a && b<c) || (b>c && b<a)) printf("Case %d: %d\n",i+1,b);
		if((c>a && c<b) || (c>b && c<a)) printf("Case %d: %d\n",i+1,c);

	}
	
	return 0;
}

File: src/UVa/11727/11727.py

'''
*****************************************************************
*                           PROBLEMA                            *
*****************************************************************
*                    UVa 11727 - Cost Cutting                   *
*****************************************************************
'''


def main():
	
	t = int(input())

	for i in range(t):

		a, b, c = map(int, input().split())

		m = a
		if (a < b and a > c) or (a > b and a < c):
			m = a
		elif (b < a and b > c) or (b > a and b < c):
			m = b
		elif (c < b and c > a) or (c < a and c > b):
			m = c

		print("Case %d: %d" % (i+1, m))


if __name__ == "__main__":
	main()
